eventstore/mongodb/eventstore.go

```diff
@@ -134,7 +134,7 @@ func (s *EventStore) Save(events []eh.Event, originalVersion int) error {
 	// Either insert a new aggregate or append to an existing.
 	if originalVersion == 0 {
 		aggregate := aggregateRecord{
-			AggregateID: aggregateID.String(),
+			AggregateID: aggregateID,
```

Comment:
This and the change below can be as before if `ID` can be used in he record above.